### ISSUE TITLE
Is the default server up?

### ISSUE THREAD

**Issue by Rreojs** [jsettlers/settlers-remake#795]
Can I play jsettlers multiplayer on my phone now?

**Comment by paulwedeck** [comment]
Yes, https://github.com/paulwedeck/settlers-remake/tree/22a0ed8a66f312452512b22efcc93e79b3aedc4d/release

1. You have to press "start server" on one phone or start the DedicatedJSettlersServer.jar on 
one desktop
2. Desktops have to add a new server entry with the local ip of the host  / Android devices (except the host) have to change the setting "server address" to the local ip of the host.
3. One player has to start a new match
4. The other must join (all players must have the map downloaded and copied to "maps" beforehand)
5. To actually start the match everybody (even the game hoster) has to press ready (red toggle button on android/red box on desktop)
6. The game hoster has to press "start game" (equal on both platforms)

**Comment by Rreojs** [comment]
When I press start server the app crashes

**Comment by paulwedeck** [comment]
Could you paste the output of logcat here?

**Comment by Rreojs** [comment]
Im sorry im on android, and there is no logcat, I press start server and sometimes I can pick the map but after that it always crashes

**Comment by paulwedeck** [comment]
You need a computer and follow this guide https://developer.android.com/studio/command-line/adb.
Connecting via wifi should be the simplest way.
You then have to run `adb logcat` in a terminal window before the step were the error occours and you will see an error message in the terminal.

**Comment by paulwedeck** [comment]
Another question: If you press "start server". Does it change to "stop server" or does it stay the same?

**Comment by Rreojs** [comment]
It just crashes

**Comment by paulwedeck** [comment]
Does it crash after you select the map or after a press "start server"?

**Comment by Rreojs** [comment]
After i select the map

**Comment by paulwedeck** [comment]
1. Does the app also crash if you select this map via Singleplayer?
2. Does the "start server" button change to "stop server" after you press it and do you see "server running" below it?

**Comment by Rreojs** [comment]
1. It works in single player
2. No nothing shows up, the app just crashes and I get back to home

**Comment by paulwedeck** [comment]
What about logcat? Are there any errors there?

**Comment by Rreojs** [comment]
Im sorry the adb programm does not launch , its too hard for me

**Comment by paulwedeck** [comment]
I added some kind of exception log.
https://github.com/paulwedeck/settlers-remake/tree/04d3bccaac10f954174fe96df3f8151e28b1adb8/release
1. Just let the app crash another time
2. restart it
3. go to settings
4. send a screenshot

**Comment by Rreojs** [comment]
Will try it out
